MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/HomographyMatrix.java

```diff
@@ -128,6 +128,47 @@ public HomographyMatrix multiplyScalar(float s) {
         return new HomographyMatrix(scaled);
     }
 
+    public static HomographyMatrix createRotationMatrix(float degrees, String axis) {
```

Comment:
Using an enum instead of String would be nicer code-wise. Any reason why you chose to use a String?

Alternative solution: have three simple functions createRotationMatrixX / createRotationMatrixY / createRotationMatrixZ, since there is very little shared code anyway.

Reply:
Separated this method out into three different functions.